ui/src/main/webapp/app/table/table-component.js

```diff
@@ -308,6 +309,24 @@ function TableController(schema, results, table, events, common, types, time, cs
         return '"' + vm.sortType + '"';
     }
 
+    vm.shouldShowTruncation = function(value) {
```

Comment:
Can you remove this as it's part of a different PR

Reply:
I'll close this PR until the table truncation PR has been merged in.